lib/content-services/src/lib/content-node-share/content-node-share.dialog.ts

```diff
@@ -154,6 +150,7 @@ export class ShareDialogComponent implements OnInit, OnDestroy {
             this.time.enable();
         } else {
             this.time.disable();
+            this.time.setValue(null);
```

Comment:
is this enough? 
the 'qshare:expiryDate' needs to be removed from the node. Does that happen?

Reply:
that code is called when dialog is being closed, you can check the `private openConfirmationDialog()` implementation